saas/api/serializers.py

```diff
@@ -407,14 +404,24 @@ def __init__(self, *args, **kwargs):
                 storage.save(name, picture)
```

Comment:
In djaodjin-signup, the `storage.save` call is done in the `APIView` class and not the serializer. This should be consistent. (I prefer saves to be done in the `APIView`.)